setup.py

```diff
@@ -97,6 +98,41 @@ def initialize_options(self):
         self.test_target = ""
 
     def run(self):
+        with open("./slack/web/client.py", "r") as original:
+            source = original.read()
+            import re
+            async_source = "\n" \
+                           "\n" \
+                           "\n" \
+                           "# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! #\n" \
+                           "#\n" \
+                           "#  *** DO NOT EDIT THIS FILE *** \n" \
+                           "#\n" \
+                           "#  Modify slack/web/client.py and run `python setup.py validate`\n" \
+                           "#\n" \
+                           "# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! #\n" \
+                           "\n" \
+                           "\n" \
+                           "\n" \
+                           + source
+            async_source = re.sub("    def ", "    async def ", async_source)
+            async_source = re.sub("from asyncio import Future\n", "", async_source)
+            async_source = re.sub("return self.api_call\(", "return await self.api_call(", async_source)
+            async_source = re.sub("Union\[Future, SlackResponse\]", "AsyncSlackResponse", async_source)
+            async_source = re.sub(
+                "from slack.web.base_client import BaseClient, SlackResponse",
+                "from slack.web.async_base_client import AsyncBaseClient, AsyncSlackResponse", async_source)
+            async_source = re.sub(
+                "class WebClient\(BaseClient\):",
+                "class AsyncWebClient(AsyncBaseClient):", async_source)
+            # from slack.web.base_client import BaseClient
+            with open('./slack/web/async_client.py', 'w') as output:
```

Comment:
Overwriting `async_client.py` with the source code based on the latest `client.py`

Reply:
Interesting approach to auto generate the async_client based off of web client